guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+
+{% include tip/close.html %}
+
+### Refunds
+
+There may be scenarios where a submarine swap is not completed successfully such as the LWS being offline. In these cases the users will need to temporarily have a refunded on-chain balance of which they can claim to an on-chain address they control. 
+
+It's recommended to keep this separate from the users main lightning balance in order to prevent confusion. 
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundOnChain@2x.png"
+   layout = "shadow"
+   caption = "If a submarine swap fails, notify the user that they need to claim a refund for the payment."
+   alt-text = "Screen showing a warning error notification prompting the user to claim a refund of their failed submarine swap."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/RefundActivity@2x.png"
+   layout = "shadow"
+   caption = "The activity page is another place where you can indicate to a user that they need to claim their refund."
+   alt-text = "Screen showing the activity menu with an entry for the user to claim their refunded payment."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/ClaimRefund@2x.png"
+   layout = "shadow"
+   caption = "Make it simple for a user to claim their refund."
+   alt-text = "Screen showing a form to claim the refund to an on-chain address."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+</div>
+
+## Received payment
+
+Once a payment is received it should be clearly indicated to the user and their balance should be updated. Ensure fees charged, if any, are clear to the user when viewing the payments details. More information on presenting payment history on our [activity]({{ '/guide/daily-spending-wallet/activity/'| relative_url }}) page. 
+
+The option to share a receipt that the payment has been received with the sender should be available and easily shared.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/requesting/received.png"
```

Comment:
I think these two success screens could go at the very top. That way we show the happy path, and then go deeper into the behind the scenes and potential problems.

Reply:
Will have a think about that some more